johnsonandjohnson/Bodiless-JS
# Feat(organisms) Carousel Image Only

## Changes
* Replaced old Slider with Carousel
* Added Carousel that in edit mode can add/remove images.
* Added Example Carousel page in examples/test-site
* Cleaned up Organism Doc

## Test Instructions

- Edit PREVIEW: Visit http://localhost:8005/carousel/  and perform the following actions:
-- First Carousel rotates manually
-- Second Carousel rotates automatically
-- Carousel is responsive and acts as expected (similar behavior to Nuka carousel demo)
- Static: Visit  http://localhost:9000/carousel/ and perform the following actions:
-- First Carousel rotates manually
-- Second Carousel rotates automatically
-- Carousel is responsive and acts as expected (similar behavior to Nuka carousel demo)
- Edit Mode: Visit http://localhost:8005/carousel/  and perform the following actions:
-- First Carousel rotates manually
-- Second Carousel does NOT rotate automatically (because we need to edit form to work)
-- Carousel is responsive and acts as expected (similar behavior to Nuka carousel demo) with the exception that the mouse slide/dragging is not enabled (because we need to edit form to work)
 -- Can Add Carousel Images 
 -- Can Delete Carousel Images
 -- Can set Images as expected (image, alt)
 - Review Docs
## Related Issues
